terraform/digitalocean/main.tf

```diff
@@ -47,7 +47,7 @@ resource "template_file" "etcd_discovery_url" {
 }
 
 module "ca" {
-  source            = "github.com/Capgemini/tf_tls//ca"
```

Comment:
Could you point the modules to github.com/tamsky/tf_tls/ for consistency which is already be used for aws and supports terraform >= 0.7.3 then we can move all back in one go to Cap